code/code/disc/disc_psionics.cc

```diff
@@ -166,7 +165,7 @@ int TBeing::doPTell(const char *arg, bool visible){
 
   mud_str_copy(capbuf, vict->pers(this), 256);  // Use Someone for tells (invis gods, etc)
 
-  char garbedBuf[256];
+  char messageBuf[256];
```

Comment:
Could we drop that `messageBuf` and instead do the string concatenation using `+` or perhaps `.format()`?

Reply:
Same as above - I used these because they were already used in the code, but I'll try moving away from them and update this weekend.